lib/models/account.js

```diff
@@ -21,6 +21,9 @@ AWSAccount.registry = new Map();
 
 exports = module.exports = AWSAccount;
 
+const accessKeyId = process.env.S3RVER_ACCESS_KEY_ID || 'S3RVER';
+const secretAccessKey = process.env.S3RVER_SECRET_ACCESS_KEY || 'S3RVER';
```

Comment:
It looks like this is being used in 2 different places. Maybe we should move these variables out into a separate file and have a single location to make the code more DRY?

OR have the version in `lib/controllers/bucket.js` reference this one.